File: exercicios/R/1060/1060.r
input = file('stdin','r')

n <- c(1:6)
c = as.integer(0)

for (i in n) {
   v = as.double(readLines(input,n=1))
   if (v > 0) {
      c = as.integer(c + 1)
   }
}
write(sprintf("%d valores positivos",c),'')

File: exercicios/JavaScript/1060/1060.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var count = 0;

for(let i=0;i<6;i++){
    var valor = lines.shift();

    if(valor > 0){
        count += 1;
    }
}
console.log(count + " valores positivos")